test/EntraBeta/Users/Set-EntraBetaUserSponsor.Tests.ps1

```diff
@@ -46,7 +46,15 @@ BeforeAll {
         }
     }
     Mock -CommandName Invoke-GraphRequest -MockWith $scriptblock -ModuleName Microsoft.Entra.Beta.Users
-    Mock -CommandName Get-EntraContext -MockWith { @{Scopes = @("User.ReadWrite.All") } } -ModuleName Microsoft.Entra.Beta.Users
+    Mock -CommandName Get-EntraContext -MockWith { 
+        @{
+            Environment="Global"
+            Scopes = @("User.ReadWrite.All") 
```

Comment:
I do see scopes again in this one. Is it a requirement specific to the Users sub-module?